azure-cli/ad/Initialize-SwaggerSPN.ps1

```diff
@@ -0,0 +1,83 @@
+function Initialize-SwaggerSPN {
```

Comment:
I think the function name should be `Initialize-SwaggerSpn`

https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)?redirectedfrom=MSDN
>you should capitalize acronyms that consist of only two characters, such as System.IO instead of System.Io.

Reply:
Done